packages/assets-controllers/src/MultichainAssetsController/index.ts

```diff
@@ -0,0 +1,10 @@
+export { MultichainAssetsController } from './MultichainAssetsController';
```

Comment:
Do we need to also add the `getDefaultMultichainAssetsControllerState` function to the list of exports?

``` suggestion
export {
  MultichainAssetsController,
  getDefaultMultichainAssetsControllerState
} from './MultichainAssetsController';
```